docs/charter.md

```diff
@@ -46,25 +51,23 @@ The _CPANSec_ intends to be a forum for coordinating and assisting in resolving
 - Share and document best security practices to authors and users;
 - Raise awareness to security threats and mitigations in Perl applications that rely on the CPAN ecosystem;
 - Act as a trusted intermediator for reporting potential security vulnerabilities to distributions, including managing and responsible disclosure of embargoed security information for a reasonable amount of time pertaining non-core distributions on CPAN, and communication with package managers and distribution channels;
-  - ~~In what ways / For how long? (e.g. max 14 days?) /  explain!~~
-    - Rationale: Let's keep specific embargo durations out of the charter, so we don't have to re-publish the charter when this changes. (garu)
 - Provide a forum for discussing and coordinating around supply chain security, integrity, metadata and the communication of these, including but not limited to:
-  - SBOM and other ways for tracking and communicating software changes, including interaction with external tooling and sources of security advisories;
+  - SBOM and other ways for tracking and communicating software and metadata changes, including interaction with external tooling and sources of security advisories;
   - The Update Framework (TUF) integration;
   - Third-party vulnerability scanning support;
 
 
-
 ### In-scope
 
 The _CPANSec_ concerns itself with a limited domain:
 
-- Distributions published on CPAN, including dual-life core modules;
-- Supply chain security, including CPAN/MetaCPAN and PAUSE
-  - [Artifact Transparency Logs](https://transparency.dev) like sigstore or sigsum
-  - Tooling and infrastructure for establishing chain-of-trust
-  - Tooling for downstream verification of chain-of-trust
-  - Tooling for the creation and verification of SBOM objects commonly in use
+- Distributions published on CPAN, not including dual-life core modules;
+- Improve various aspects of CPAN supply chain security, including the Perl/CPAN toolchains, CPAN/MetaCPAN itself and PAUSE;
```

Comment:
I think wording should be changed either on the list or on the sentence right above the list. The latter refers to a "limited domain", but the list was changed to start with verbs, which indicate actions, not spaces.

Reply:
Aah, good point. I'll reformulate the points below.